src/index.html

```diff
@@ -4,14 +4,187 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      class="form"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field" for="user_surname">
+          Surname:
+          <input type="text"
+          id="user_surname"
+          name="user_surname"
+          minlength="2"
+          maxlength="50"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="user_surname"
            name="user_surname"
            minlength="2"
            maxlength="50"
            autocomplete="off"
            required
          >
```
All attributes should be on the new lines in this case and have two spaces before them.
Earlier I've left a link when you can read about it